src/create-jobs.js

```diff
@@ -78,10 +90,11 @@ const createPHPUnitJobs = function (config) {
  * @return {Job}
  */
 const createPHPUnitJob = function (version, deps, config) {
+    const fileName = appendDistExtensionIfFilenameDoesntExist('phpunit.xml');
     return new Job(
         'PHPUnit on PHP ' + version + ' with ' + deps + ' dependencies',
         JSON.stringify(new Command(
-            './vendor/bin/phpunit',
+            `xmllint --noout --schema ${xmlSchema.phpunit} ${fileName} && ./vendor/bin/phpunit`,
```

Comment:
Why `--noout`? It seems like we may want to see _why_ it failed.

Reply:
The docs suggest this only suppresses output, not errors, but I will remove anyway.